.github/workflows/mingw-w64.yml

```diff
@@ -0,0 +1,65 @@
+name: mingw-w64
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        with-patch:
+          - true
+          - false
+    env:
+      MINGW_PACKAGE_PREFIX: mingw-w64-ucrt-x86_64
+      MSYSTEM: UCRT64
```

Comment:
This environment variable is required. `bash -l` (using bash as a login shell) refers this environment variable and setup `PATH` and so on.